packages/cron-jobs/src/jobs/update-checker.ts

```diff
@@ -0,0 +1,58 @@
+import { Octokit } from "octokit";
+import { compareSemVer, isValidSemVer } from "semver-parser";
+
+import { EVERY_HOUR } from "@homarr/cron-jobs-core/expressions";
+import { logger } from "@homarr/log";
+
+import packageJson from "../../../../package.json";
+import { createSubPubChannel } from "../../../redis/src/lib/channel";
+import { createCronJob } from "../lib";
+
+export const updateCheckerJob = createCronJob("updateChecker", EVERY_HOUR, {
```

Comment:
What about using a `request handler` instead?

Reply:
All other cron jobs for non-integrations do not use it. Why should we here? Isn't it exclusively for the integrations?

Reply:
Your correct, right now we only use it with the cached-integration-request-handler, but I build it that way that all requests can be handled centralized and cached. For example my plan is to move the ping results to this system as well. You can simply use the createCachedRequestHandler method provided